README.md

```diff
@@ -25,3 +26,41 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+**My Explanation:**
+
+Recurrence Relation:
+
+T(n) = {1 if n <= 2, 3T(n/3) + n
```

Comment:
...so you're doing $n$ work to combine the partial results?

Reply:
It takes n time to slice the array, does it not? Dividing the array into 3 parts during recursion means log(n) time. And calculating the sum ultimately takes n time, does it not?

Reply:
How many operations are you doing to combine the partial results?